libbeat/docs/processors-using.asciidoc

```diff
@@ -761,3 +762,33 @@ The fields added to the event are looking as following:
 -------------------------------------------------------------------------------
 
 NOTE: The host information is refreshed every 5 minutes.
+
+[[dissect]]
+=== Dissect
```

Comment:
We're trying to use goal-oriented titles here. How about:

Dissect strings or Tokenize strings?

Reply:
I will go with `Dissect strings`